data/technical_steering_committee.json

```diff
@@ -40,5 +40,26 @@
         "gitHubAccount": "https://github.com/Perseverance",
         "photo": "george.jpg",
         "bio": "George Spasov is a blockchain architect and co-founder of LimeChain. As part of LimeChain, over the past 8 years he has overseen the ideation, architecture creation, development and release of tens of projects including blockchain networks, L2 rollups, decentralized applications, infrastructure and customer facing products. George brings a vast knowledge of both Hedera and the wider DLT space."
+    },
+    {
+        "firstName": "Milan",
+        "lastName":"Wiercx van Rhijn",
+        "gitHubAccount": "https://github.com/MilanWR",
+        "photo": "milan.jpeg",
+        "bio": "TBD"
+    },
+    {
+        "firstName": "Michael",
+        "lastName":"Kantor",
+        "gitHubAccount": "https://github.com/kantorcodes",
+        "photo": "michael.jpeg",
+        "bio": "TBD"
+    },
+    {
+        "firstName": "Brandon",
+        "lastName":"Davenport",
+        "gitHubAccount": "https://github.com/itsbrandondev",
+        "photo": "brandon.jpeg",
+        "bio": "TBD"
```

Comment:
```suggestion
        "bio": "Brandon Davenport is Director of Communications at Hgraph, a leading infrastructure provider and technical partner for enterprise, startups and developers. He pairs 15 years of content and brand leadership with hands-on open-source governance, product development, and community building. A major Hedera enthusiast since 2019, Brandon champions open-source and startup collaboration across the entire ecosystem. Based in Ottawa, he enjoys running, playing music, and spending time with his wife and their pets."
```